### PR TITLE
Announcements for 8.7 release

### PR DESCRIPTION
**Pull request by mesellings**
## Description

Announcements for the 8.7 release.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [X] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [X] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by mesellings** [comment]
Hi @felix-mueller Here is the first draft of the Announcements page(s):

https://preview.docs.camunda.cloud/pr-4679/docs/next/reference/announcements/

Please review and also tag/bring in anyone who might be able to help with adding more technical content for the release - I've added in content where I could find it/figure it out, but there might be more detail needed, e.g. in the Exporters/deployment sections?

Regarding the blogs, we can hide these links until they are published, and then add them in.

As mentioned in the Slack channel, I incorporated some restructuring around the Announcements section to provide a separate announcements page for each version. The main landing page is quite bare for now, but can be jazzed up later once we have the 8.7 page completed.

FYI @akeller @MaxTru

File: docs/reference/announcements/870.md
Comment (MaxTru) [inline docs/reference/announcements/870.md:9]: I suggest we don't call it `What's coming in Camunda 8.7` because this reads like a big overlap to https://docs.camunda.io/docs/next/reference/release-notes/870/, which features `New features in 8.7.0`.

Maybe we just leave this away, since the previous line is already clear enough?
> Important changes and updates for the Camunda 8.7 release that customers should be aware of are summarized below.

File: docs/reference/announcements/870.md
Comment (theburi) [inline docs/reference/announcements/870.md:43]: Could we add 'v1' API qualifier as it reads that we deprecate all Operate, Tasklist API's

File: docs/reference/announcements/870.md
Comment (theburi) [inline docs/reference/announcements/870.md:59]: Can we rephrase to clarify and avoid confusion?
"With Camunda 8.7 release, we will announce depreciation of several grps endpoints that will be removed in 8.9."
It could be read in to that 8.7 already remove some endpoints or even all grps endpoints.

File: docs/reference/announcements/870.md
Comment (theburi) [inline docs/reference/announcements/870.md:67]: Can we publish the date of removal?

File: docs/reference/announcements/870.md
Comment (theburi) [inline docs/reference/announcements/870.md:108]: Could we add another section:
### Camunda 8.7 manual installation
For organizations that do not use cloud-native platforms like Kubernetes or container services, we will publish a reference architecture that provides guidance on implementing Camunda production clusters on VM-based systems, using AWS EC2 as an example. The architecture will include details on optimal instance sizing, network configurations, and security best practices to ensure robust performance and reliability.

File: docs/reference/announcements/870.md
Comment (mesellings) [reply docs/reference/announcements/870.md:9]: Agreed, will remove 👍

File: docs/reference/announcements/870.md
Comment (mesellings) [reply docs/reference/announcements/870.md:57]: Added 👍

File: docs/reference/announcements/870.md
Comment (mesellings) [reply docs/reference/announcements/870.md:65]: @theburi We can, but do we even know this? If so, who might be best placed to answer this?

File: docs/reference/announcements/870.md
Comment (mesellings) [reply docs/reference/announcements/870.md:106]: Added 👍

File: docs/reference/announcements/870.md
Comment (akeller) [reply docs/reference/announcements/870.md:65]: @aleksander-dytko

**Review by akeller — DISMISSED** [review]
🔧 We can entirely remove references to 8.2, as it's unsupported/unmaintained. Please remove them from this PR.

I like that we include the release date on the announcements overview page, but I'm concerned about changing dates in multiple places. Will you add this to the release manager tasks @mesellings about where to update these dates?

File: docs/reference/announcements/870.md
Comment (akeller) [inline docs/reference/announcements/870.md:26]: Need the links here.

File: docs/reference/announcements/870.md
Comment (akeller) [reply docs/reference/announcements/870.md:65]: Related to https://github.com/camunda/camunda-docs/pull/4649

Comment (akeller) [reply docs/reference/announcements/870.md:53]: @marcosgvieira can you help us here? Can we add a date or a version for removal?

File: docs/reference/announcements/870.md
Comment (hisImminence) [inline docs/reference/announcements/870.md:93]: @hamza-m-masood / @aabouzaid should we add the removal of the separate ingress here already? Its already merged and will be released with alpha2, right?

**Comment by mesellings** [comment]
FYI @ChrisKujawa @akeller I've added the content from https://github.com/camunda/camunda-docs/pull/4715 to this PR now (including your review comments @akeller thanks)

File: docs/reference/announcements/870.md
Comment (aabouzaid) [reply docs/reference/announcements/870.md:93]: @hisImminence Added :heavy_check_mark:

Comment (hisImminence) [reply docs/reference/announcements/870.md:93]: Thank you!

**Comment by mesellings** [comment]
Moved out of draft status in anticipation of release.
- @akeller @felix-mueller did you want to conduct a final review? Anyone else you think should also?
- Is this to be released in alignment with the release next Tuesday, or on a separate timeline?
- The only outstanding item I'm aware of now is whether @marcosgvieira can supply wus with a date for the removal of the Tasklist GraphQL APi (https://github.com/camunda/camunda-docs/pull/4679#discussion_r1870333231)

File: docs/reference/announcements/870.md
Comment (akeller) [reply docs/reference/announcements/870.md:53]: @mesellings see https://docs.google.com/document/d/1aqwSXP8uF3I7cV6Rz-HmBYIXSltwlmfjHKmmlVWBg6o/edit?tab=t.0#heading=h.zhyuwlliwpqj

We can announce this was (is?) removed in 8.7, but I'm still not sure if we can associate it with an alpha. For the date, we could set it for the minor release date at the latest (Feb 11).

**Comment by akeller** [comment]
> Is this to be released in alignment with the release next Tuesday, or on a separate timeline?

@mesellings Unless we have announcements specific to an alpha, they usually go out with the minor release. Given that we have such impactful changes, some already realized in alphas, I would recommend merging and iterating on this between now and the 8.7 release. I would like input from @MaxTru and @felix-mueller as stakeholders for this decision.

File: docs/reference/announcements/870.md
Comment (mesellings) [reply docs/reference/announcements/870.md:53]: Thanks, I've amended the entry, I think we can close this conversation now 👍

**Comment by mesellings** [comment]
> 🔧 We can entirely remove references to 8.2, as it's unsupported/unmaintained. Please remove them from this PR.
> 
> I like that we include the release date on the announcements overview page, but I'm concerned about changing dates in multiple places. Will you add this to the release manager tasks @mesellings about where to update these dates?

Added to: https://confluence.camunda.com/display/HAN/Docs+Release+Rituals

**Review by MaxTru — APPROVED** [review]
Good job! 

IMO we can already merge and release this - it is an important heads-up for our users. So the earlier we release this, the more time they have to digest this information.

**Comment by mesellings** [comment]
Merging and releasing so we can get this info out to customers - we can follow up with any further modifications in another PR if necessary.

**Comment by mesellings** [comment]
FYI @MaxTru @akeller This is now live: https://docs.camunda.io/docs/next/reference/announcements/announcements-870/

**Pull request merged**
